pkg/types/gcp/machinepools.go

```diff
@@ -33,6 +33,11 @@ type OSDisk struct {
 	// +kubebuilder:validation:Minimum=16
 	// +kubebuilder:validation:Maximum=65536
 	DiskSizeGB int64 `json:"DiskSizeGB"`
+
+	// EncryptionKey defines the kKMS key to be used to encrypt the disk
```

Comment:
The field descriptions should end with periods.
```suggestion
	// EncryptionKey defines the kKMS key to be used to encrypt the disk.
```